README.md

```diff
@@ -141,6 +142,17 @@ test[0].bar.x = 2;
 type SimplifiedComplexObject = Omit<ComplexObject, "nested">;
 ```
 
+### OmitProperties
+
```

Comment:
Could you leave 1 sentence description? What does it do? Something like Removes all properties extending type P in type T.